hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -145,6 +145,8 @@
                 export REGISTER_MASTER="true"
                 # Reduce logs verbosity
                 export TEST_CLUSTER_LOG_LEVEL="--v=2"
+                # TODO: Remove when we figure out #22166 and other docker potential slowness
+                export DOCKER_TEST_LOG_LEVEL="--log-level=info"
```

Comment:
This is weird. On my local branch, it is totally gone. But remote branch doesn't have that change. Let me rebase and repush again.